packages/inferno-test-utils/__tests__/testUtils.jest.spec.jsx

```diff
@@ -0,0 +1,20 @@
+import { render } from "inferno";
+import Component from "inferno-component";
+import createClass from "inferno-create-class";
+import createElement from "inferno-create-element";
+import TestUtils from "inferno-test-utils";
+
+const FunctionalComponent = function(props) {
+  return createElement("div", props);
+};
+
+describe("renderToSnapshot", () => {
+  it("should return a snapshot from a valid vNode", () => {
+    const snapshot = TestUtils.renderToSnapshot(
+      <FunctionalComponent foo="bar" />
+    );
+    expect(snapshot.props).toHaveProperty("foo");
```

Comment:
This will need to become `expect(snapshot.props.foo).toBeDefined()`, due to using "vanilla" karma in our browser tests. I haven't thought of a nice way to differentiate between browser and jest tests yet.